installer/PowerToysSetup/Product.wxs

```diff
@@ -1090,6 +1097,17 @@
         <?endforeach?>
       </Component>
 
+      <!-- UnitConverter Plugin -->
+      <Component Id="UnitConverterComponent" Directory="UnitConverterPluginFolder" Guid="D4F429E3-C619-49D6-9416-88A757D18E02">
+        <?foreach File in plugin.json;Community.PowerToys.Run.Plugin.UnitConverter.dll;ManagedTelemetry.dll?>
```

Comment:
Add `Community.PowerToys.Run.Plugin.UnitConverter.deps.json`

Reply:
[Added. ](https://github.com/microsoft/PowerToys/pull/11617/commits/396b211adb01addb14986f57ee8a07564766cecd#diff-668b4388b55bb934d7ceccbfdd172f69257c9c607ca19cb9752d4a4940b69886R1102)
Does `ManagedTelemetry.dll` need to be removed?

Reply:
Why do you want to delete `ManagedTelemetry.dll`?

Reply:
I don't see it included in the VSCodeworkspace plugin entry, and I didn't do anything else for telemetry either. Not entirely sure if it's correct to have it here without a 'full'(?) implementation.
I tried searching the wiki but couldn't find much about it.

I figured it might be best to remove it for the while being, until [this](https://github.com/microsoft/PowerToys/pull/11406#issuecomment-847712995) is done.

Reply:
Okay, then delete it.

Reply:
deleted it